Gopkg.toml

```diff
@@ -116,3 +116,7 @@
 [[constraint]]
   name = "github.com/gin-contrib/cors"
   version = "1.2.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Is there a version tag we could pin this to?

Reply:
@j16r they don't have any tags in that project unfortunately. Is the SHA revision lock enough? `revision = "4fbfcc6122759afed2efbc762c4c41164a403951"`

Reply:
Yeah that would be better.